Model/Sync/Reset/Customers.php

```diff
@@ -34,13 +36,26 @@ public function getYotpoEntityName()
 
     /**
      * @param int $storeId
-     * @param boolean $skipSyncTables
+     * @param boolean $clearSyncTables
      * @return void
      * @throws \Zend_Db_Statement_Exception
      */
-    public function resetSync($storeId, $skipSyncTables = false)
+    public function resetSync($storeId, $clearSyncTables = true)
     {
         parent::resetSync($storeId);
+        $this->resetCustomerSyncAttributes();
         $this->setResetInProgressConfig($storeId, '0');
     }
+
+    /**
+     * @return void
+     */
+    private function resetCustomerSyncAttributes()
+    {
+        $this->updateEntityAttributeTableData(
+            0,
```

Comment:
I understand that passing store ID 0 should represent a flag.
This is an approach that is violating clean code, let's separate the method `updateEntityAttributeTableData` if needed.

Reply:
I  have created a new function for updateEntityAttributeTableDataWithoutStoreId . Please check the updated PR